src/stories/StyledCoinListItem.tsx

```diff
@@ -0,0 +1,43 @@
+import DfxAssetIcon, { AssetIconVariant } from './DfxAssetIcon';
+
+export interface StyledCoinListItemProps {
+  asset: AssetIconVariant;
+  onClick: () => void;
+  protocol: Protocols;
+}
+
+export enum Protocols {
+  ERC_20 = 'ERC-20',
+  BEP_20 = 'BEP-20',
+}
+
+const NAME_MAPS: Record<AssetIconVariant, string> = {
+  [AssetIconVariant.USDT]: 'USD Tether',
+  [AssetIconVariant.BNB]: 'BNB',
+  [AssetIconVariant.DFI]: 'DFI',
+  [AssetIconVariant.USDC]: 'USD Coin',
+  [AssetIconVariant.BUSD]: 'Binance USD',
+  [AssetIconVariant.ETH]: 'Ethereum',
+  [AssetIconVariant.DAI]: 'Dai',
+  [AssetIconVariant.BTC]: 'Bitcoin',
+};
```

Comment:
Would it make sense to set the names directly on enum definition? Like
```ts
export enum AssetIconVariant {
  USDT = 'USD Tether',
  BNB = 'BNB',
  DFI = 'DFI',
  USDC = 'USD Coin',
  BUSD = 'Binance USD',
  ...
}
```

Reply:
generally true, tried it, it messes up the fallback situation. 
for now i would leave it as is, and maybe refactor the whole thing with  a more dynamic fallback icon post-beta...